### DISCUSSION TITLE
MoreCulling - v0.6.0-pre1

### DISCUSSION THREAD

**Discussion by FxMorin** [FxMorin/MoreCulling#40] [General]
### This is a pre-release! - Please give feedback in the discussion  

This update will be adding Leaves Culling multiple different types. I want your opinions on it.  

**Culling Info:**  
4 different ways to cull leaves.  
- Fast (Ugly but 3.35x faster than vanilla)  
- BlockState (Decent, 1.76x faster than vanilla)  
- Check Surrounding (Good, 1.39x faster than vanilla)  
- Depth (Has 4 modes, ranges from 2.67-1.22x faster than vanilla)  - Higher is Faster  

<hr /><em>This discussion was created from the release <a href='https://github.com/fxmorin/MoreCulling/releases/tag/v0.6.0-pre1'>MoreCulling - v0.6.0-pre1</a>.</em>

**Comment by DoTrONoL** [comment]
**Just asking.**
Why perform a new Leaves Culling if we do have "Cull Less Leaves"? Your method is "better" than they? And besides that, would it cause a conflict with them? ~Judging that you can do a different method, I wonder if they could perform together.~

Well, in the final, if your mod can get me more FPS than I'm migrating to it. Haha. 😉👊

Thanks by the new update. Cheers!

**Reply by FxMorin** [reply]
So I'm using 4 unique methods, which is why I'm making my own. 

Currently if you are using CullLessLeaves, my implementation will not work. I prioritize CullLessLeaves, since some people might like the look of those leaves more. While others might like mine more.
I like to give people options.

With leaves culling, it's all about how it looks to you, while not looking like fast, so it's up to you to make a decision
